File: Frontend/CRUD/Modelos/LogCrud.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CRUD.Modelos
{
    /// <summary>
    /// Clase para los datos de log de errores.
    /// </summary>
    public class LogCrud
    {
        public String IdLog { get; set; }
        public String Sistema { get; set; }
        public String Metodo { get; set; }
        public String Tipo { get; set; }
        public String Descripcion { get; set; }
        public DateTime Fecha { get; set; }
    }
}

File: Backend/src/main/java/com/devsrcoppel/crud/models/LogCrud.java
package com.devsrcoppel.crud.models;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@Getter
@Setter
public class LogCrud {
    @JsonProperty("IdLog")
    private String IdLog;
    @JsonProperty("Sistema")
    private String Sistema;
    @JsonProperty("Metodo")
    private String Metodo;
    @JsonProperty("Tipo")
    private String Tipo;
    @JsonProperty("Descripcion")
    private String Descripcion;
    @JsonProperty("Fecha")
    private Date Fecha;

    public LogCrud() {

    }
}
